### PR TITLE
Add git autoupdate config in hasher

### PR DESCRIPTION
**Pull request by Piicksarn**

### CODE REVIEW

**Comment by PeterDaveHello** [comment]
Wrong config.

**Comment by Piicksarn** [comment]
is it ok now?~

**Comment by PeterDaveHello** [comment]
Still wrong.

**Comment by Piicksarn** [comment]
have it corrected! thank you!

File: ajax/libs/hasher/package.json [hunk 1]
```diff
@@ -22,5 +31,8 @@
...
+  "_id": "hasher@1.2.0",
+  "_from": "hasher"
```
Comment (PeterDaveHello) [inline ajax/libs/hasher/package.json]: What's the reason why to add these lines here?

**Comment by Piicksarn** [comment]
just a mistake!
corrected it!

**Comment by PeterDaveHello** [comment]
no you **didn't**

**Comment by Piicksarn** [comment]
sorry! It's fine now

**Pull request merged**